raiden/ui/cli.py

```diff
@@ -1055,12 +1061,19 @@ def stop_task(task):
                 finally:
                     task.get()  # re-raise
 
-            stop_tasks = [gevent.spawn(stop_task, task) for task in tasks]
-            gevent.joinall(
-                stop_tasks,
-                app_.config.get('shutdown_timeout', DEFAULT_SHUTDOWN_TIMEOUT),
-                raise_error=True,
-            )
+            try:
```

Comment:
Don't handle here, this is exactly the point where any unhandled exceptions **until this point** should re-raise

Reply:
the exception was not handled until here, where do you want it to be handled?

Reply:
[Here](https://github.com/raiden-network/raiden/pull/2365#discussion_r215314556)

Reply:
done by https://github.com/raiden-network/raiden/pull/2446